tsconfig.strict.json

```diff
@@ -90,6 +91,7 @@
   ],
   "exclude": [
     "lib/constants/platform.spec.ts",
+    "lib/datasource/adoptium-java/**/*.spec.ts",
```

Comment:
Too hard?

Reply:
Usually I try to include test files as well, but in this case it transitively depends on the entire `lib/versioning/` directory.

Reply:
Ok, so we should fix versionings first.

Reply:
Also it depends on `lib/datasource/index.ts`, so it seems to be a circular situation. My reasoning behind listing `*.spec.ts` files in the `exclude` section is that their null errors are mostly fixed locally, i.e. once all non-test files are refactored, spec files from `exclude` section should work automatically.